tests/unit/test_s3.py

```diff
@@ -1015,3 +1017,23 @@ def test_s3_aws_chunked_decoder_chunk_bigger_than_s3_chunk(self):
         assert stream.read(chunk_size + 1000) == total_body[:chunk_size]
         # assert that even if we read more, when accessing the rest, we're still at the same position
         assert stream.read(10) == total_body[chunk_size : chunk_size + 10]
+
+
+class TestS3TemporaryStorageBackend:
+    def test_get_fileobj_no_bucket(self):
+        temp_storage_backend = EphemeralS3ObjectStore()
```

Comment:
as suggested in the `EphemeralS3ObjectStore`, it would be great if we could use the `tmp_path` or `tmpdir` fixture and inject it into the class instead of using a global tmp directory.